app/js/LocalController.js

```diff
@@ -79,26 +79,22 @@ angular.module('makeithappen').controller('LocalController', ['$scope', function
         "items": $scope.schema
     };
     $scope.usersUiSchema = {
-        "elements": [
+        "type": "Control",
+        "scope": {
+            "$ref": "#"
+        },
+        "columns": [
             {
-                "type": "Control",
```

Comment:
The elements in the columns do not need "type":"Control" ? Maybe we should support any type of element there?